client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/index.tsx

```diff
@@ -312,18 +358,21 @@ const designSetup: Step = function DesignSetup( { navigation, flow } ) {
 
 	stepContent = showGeneratedDesigns ? (
 		<GeneratedDesignPicker
-			designs={ designs }
+			selectedDesign={ selectedDesign || shuffledGeneratedDesigns[ 0 ] }
```

Comment:
I think this makes the first one always be selected on the mobile
![image](https://user-images.githubusercontent.com/13596067/167084058-b2702246-df73-4d08-9f05-7cdfc802106a.png)